sdk/lib/atl/atlwin.h

```diff
@@ -320,9 +320,37 @@ class CWindow
         int wndCenterHeight = wndCenterRect.bottom - wndCenterRect.top;
         int wndWidth = wndRect.right - wndRect.left;
         int wndHeight = wndRect.bottom - wndRect.top;
+        int xPos = wndCenterRect.left + ((wndCenterWidth - wndWidth + 1) >> 1);
+        int yPos = wndCenterRect.top + ((wndCenterHeight - wndHeight + 1) >> 1);
+
+        if (!(::GetWindowLong(hWndCenter, GWL_STYLE) & WS_CHILD))
+        {
+            MONITORINFO m_MonInfo;
+            m_MonInfo.cbSize = sizeof(MONITORINFO);
```

Comment:
```suggestion
            m_MonInfo.cbSize = sizeof(m_MonInfo);
```
Also the notation `m_` is reserved to members of the class, but here, the monitor info is just a local variable, so please avoid using it.

Reply:
Have you seen the discussion about the `m_` prefix?

Reply:
NO

Reply:
I fixed it.